.buildkite/bench-api.sh

```diff
@@ -8,11 +8,8 @@ set -euo pipefail
 # see also <https://stackoverflow.com/a/63751678>.
 export LC_ALL=C.UTF-8
 
-if [ -n "${SCRATCH_DIR:-}" ]; then
-  mkdir -pv "$SCRATCH_DIR"
-  export TMPDIR="$SCRATCH_DIR/tmp"
-  mkdir -pv "$TMPDIR"
-fi
+export TMPDIR="/tmp/bench/api"
+mkdir -p $TMPDIR
```

Comment:
```suggestion
export TMPDIR="$TMPDIR/bench/api"
mkdir -p $TMPDIR
```

Without the shell interpolation, this will likely fail on macOS machines, because `/tmp` is not the canonical path for temporary directories on macOS. Example:

```
$ echo $TMPDIR
/var/folders/g7/7du81ti_b7mm84n184fn3k910000lg/T/
```